src/lib/progress-spinner/progress-spinner.ts

```diff
@@ -25,6 +25,9 @@ const endIndeterminate = 80;
 /* Maximum angle for the arc. The angle can't be exactly 360, because the arc becomes hidden. */
 const MAX_ANGLE = 359.99 / 100;
 
+// Stroke width is by default 10px, which is relatively to the viewBox size 10%.
```

Comment:
Not sure whether this comment is very helpful. We'd also need to keep it updated if we change the default value. Something like "Default stroke width as a percentage of the viewBox" might be better.